airflow/cli/commands/plugins_command.py

```diff
@@ -78,33 +75,17 @@ def dump_plugins(args):
         print("No plugins loaded")
         return
 
-    console = Console()
-    console.print("[bold yellow]SUMMARY:[/bold yellow]")
-    console.print(
-        f"[bold green]Plugins directory[/bold green]: {conf.get('core', 'plugins_folder')}\n", highlight=False
-    )
-    console.print(
-        f"[bold green]Loaded plugins[/bold green]: {len(plugins_manager.plugins)}\n", highlight=False
-    )
+    plugins_info: List[Dict[str, str]] = []
+    for plugin in plugins_manager.plugins:
+        info = {"name": plugin.name}
+        info.update({n: getattr(plugin, n) for n in PLUGINS_ATTRIBUTES_TO_DUMP})
+        plugins_info.append(info)
 
-    for attr_name in PLUGINS_MANAGER_ATTRIBUTES_TO_DUMP:
-        attr_value: Optional[List[Any]] = getattr(plugins_manager, attr_name)
-        if not attr_value:
-            continue
-        table = SimpleTable(title=attr_name.capitalize().replace("_", " "))
-        table.add_column(width=100)
-        for item in attr_value:  # pylint: disable=not-an-iterable
-            table.add_row(
-                f"- {_get_name(item)}",
-            )
-        console.print(table)
+    # Remove empty info
+    if args.output == "table":  # pylint: disable=too-many-nested-blocks
+        for col in list(plugins_info[0].keys()):  # it will exist as there's at least one plugin
```

Comment:
```suggestion
        for col in plugins_info[0]:  # it will exist as there's at least one plugin
```

Reply:
And I'm not sure if the comment is correct. I assume it's possible to have zero plugin?

Reply:
The list should stay there. It's to make sure we iterate over fixed range - the dict may change over time due to `del` statement.

Reply:
Ja, correct. My bad.

How about the comment? May you clarify?

Reply:
Extended the comment. It was mostly about the `xs[0]` as normally the element my not exists, but in this case we are sure there will be at least on plugin - we check it few lines earlier.